AnnoyancesFilter/sections/popups.txt

```diff
@@ -113,6 +113,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt ######### 
 !
+pixnet.net##.idle-pop
```

Comment:
Not reproduced on my end and I don't see the element on the code. Do you have problems scrolling the page if you block the element?
Btw, the ad element should be in the `AdGuard Chinese - specific.txt`.

Reply:
Sorry for the late response.
The element only pop up if you leave the page idle for more than 3 minutes.

![image](https://user-images.githubusercontent.com/9109405/141323534-a1228330-a56c-40bf-8573-4e4346767a57.png)

Reply:
now I can't reproduce too. maybe they changed something.

Reply:
Maybe. If you find it again, just let me know )